gdsc-web-backend/Controllers/PagesController.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using gdsc_web_backend.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace gdsc_web_backend.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class PagesController:ControllerBase
```

Comment:
The code is not properly formatted, you must have spaces around inheritance colons :(((